zerver/models.py

```diff
@@ -723,6 +739,21 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
     # they will not be available regardless of users' personal settings.
     enable_read_receipts = models.BooleanField(default=False)
 
+    realm_permission_group_settings: Dict[str, GroupPermissionSetting] = {
```

Comment:
Maybe we should all-caps this name, since that's our convention for data hanging off the class that is constant configuration, not a database field.

@sahil839 thoughts? By that reasoning, we should possibly be renaming `property_types` to `PROPERTY_TYPES`, and similar; we definitely shouldn't do that in this PR.

Reply:
Yes, sounds good.